composer.json

```diff
@@ -8,13 +8,12 @@
     "prefer-stable": true,
     "require": {
         "composer-plugin-api": "^2.0",
-        "php": ">=8.0"
+        "php": ">=8.1"
```

Comment:
Should go directly 8.2 as lowest supported version as 8.1 eol is by the end of this year so we don't have to update again?